Types for xray-settings and report-config are not shipped with library

When compiling project with `tsc` I get error:

```
node_modules/@xray-app/xray-automation/dist/types/index.d.ts:6:35 - error TS2307: Cannot find module '../types/xray-settings' or its corresponding type declarations.

6 import type { XraySettings } from "../types/xray-settings";
                                    ~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@xray-app/xray-automation/dist/types/index.d.ts:7:35 - error TS2307: Cannot find module '../types/report-config' or its corresponding type declarations.

7 import type { ReportConfig } from "../types/report-config";
                                    ~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@xray-app/xray-automation/dist/types/xray-datacenter-client.d.ts:1:35 - error TS2307: Cannot find module '../types/xray-settings' or its corresponding type declarations.

1 import type { XraySettings } from "../types/xray-settings";
                                    ~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@xray-app/xray-automation/dist/types/xray-datacenter-client.d.ts:2:35 - error TS2307: Cannot find module '../types/report-config' or its corresponding type declarations.

2 import type { ReportConfig } from "../types/report-config";
                                    ~~~~~~~~~~~~~~~~~~~~~~~~


Found 4 errors in 2 files.

Errors  Files
     2  node_modules/@xray-app/xray-automation/dist/types/index.d.ts:6
     2  node_modules/@xray-app/xray-automation/dist/types/xray-datacenter-client.d.ts:1
```

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "build:cjs": "node tools/cleanup cjs && tsc -p config/tsconfig.cjs.json",
     "build:esm": "node tools/cleanup esm && tsc -p config/tsconfig.esm.json",
     "build:umd": "node tools/cleanup umd && webpack --config config/webpack.config.js",
-    "build:types": "node tools/cleanup types && tsc -p config/tsconfig.types.json",
+    "build:types": "node tools/cleanup types && tsc -p config/tsconfig.types.json && cp types/xray-settings.d.ts dist/types/xray-settings.d.ts && cp types/report-config.d.ts dist/types/report-config.d.ts",
     "clean": "node tools/cleanup",
     "package": "npm run build && npm pack",
     "prepare": "husky install && npm run build",

